requirements.txt

```
@@ -0,0 +1,3 @@
+numpy==1.16.2
+opencv_python==4.1.0.25
```

on linux: 
```
ERROR: Could not find a version that satisfies the requirement opencv_python==4.1.0.25 (from -r requirements.txt (line 2)) (from versions: 3.4.8.29, 4.1.2.30)
```
So I suggest putting  opencv_python==4.1.0.25 
```suggestion
opencv_python>=4.1.0
```